ci/pipelines/pull-request/jinja.template.yml

```diff
@@ -21,27 +21,20 @@
 groups:
 - name: main
   jobs:
-  - Build
 {%- for test in tests if not test.name.startswith("Windows") %}
   - {{test.name}}Test
 {%- endfor %}
 resources:
-- name: docker-geode-build-image
-  type: docker-image
-  source:
-    password: ((!docker-password))
-    repository: gcr.io/((gcp-project))/((pipeline-prefix))((!docker-image-name))
-    tag: latest
-    username: ((!docker-username))
 - name: geode
   type: pull-request
   source:
     access_token: ((!github-pr-access-token))
-    base: develop
```

Comment:
Not that it is really a problem, but does the new resource implicitly run `precheckin` against any PR that is opened, for instance, if we backport a fix onto `release/1.7.0`?

Reply:
the PR metadata encodes what the base is, so its not needed in YML anymore. This is an artifact of the migration from GitHub APIv3 vs v4